src/amo/components/UserProfileEdit/index.js

```diff
@@ -549,104 +551,102 @@ export class UserProfileEditBase extends React.Component<Props, State> {
                 value={this.state.displayName}
               />
 
-              {/*
-                TODO: Don't show these to users who don't have a public-facing
-                user profile page (eg are developers). It's just noise and may
-                encourage them to enter a lot of text (especially the bio) which
-                no one will see. It also gets in the way of settings,
-                like notifications, below.
-                See: https://github.com/mozilla/addons-frontend/issues/4964
-              */}
-              <label className="UserProfileEdit--label" htmlFor="homepage">
-                {i18n.gettext('Homepage')}
-              </label>
-              <input
-                className="UserProfileEdit-homepage"
-                disabled={!user}
-                id="homepage"
-                name="homepage"
-                onChange={this.onFieldChange}
-                type="url"
-                value={this.state.homepage}
-              />
-              <p className="UserProfileEdit-homepage--help">
-                {i18n.gettext(`This URL will only be visible for users who are
-                  developers.`)}
-              </p>
-
-              <label className="UserProfileEdit--label" htmlFor="location">
-                {i18n.gettext('Location')}
-              </label>
-              <input
-                className="UserProfileEdit-location"
-                disabled={!user}
-                id="location"
-                name="location"
-                onChange={this.onFieldChange}
-                value={this.state.location}
-              />
+              {isDeveloper(user) && (
+                <React.Fragment>
+                  <label className="UserProfileEdit--label" htmlFor="homepage">
```

Comment:
is it possible that these fields could already set up by some users? for example even a picture which maybe a regular user might add..
I am just wondering if we should do anything to hide these on the user page too (outside of edit) since now they won't be able to change this with these changes, right? This may be an edge case or not possible tho...do you know?